### ISSUE TITLE
Recommended bitrate / options for x265/HEVC?

### ISSUE THREAD

**Issue by jdelman** [donmelton/video_transcoding#216]
I recently encoded a 1080p Blu-ray using the `--handbrake-option encoder=x265` option. It produced an MKV that was the same exact file size as when I had run it earlier without that option. It's my understanding that you can use a much lower bitrate with x265 but achieve similar results.

What is a good target bitrate for x265 that would match the visual quality of iTunes? What arguments should I pass to `video-transcoding`?

Thanks!

**Comment by donmelton** [comment]
@jdelman Yes, the output file size would be roughly the same no matter whether you use `x264` or `x265`. That's because the target bitrate does not change. However, in most situations, quality will improve when using `x265` at the same bitrate as `x264`.

To reduce the bitrate, try adding `--target small` to your command line.

I don't think I've done sufficient testing to provide guidance on options to match the visual quality of iTunes when using `x265`. That's an "eye of the beholder" kind of thing anyway. :) And, for me, `x265` is still too slow to be practical for home transcoding.

I hope that helps.
